docs/sources/write/links/index.md

```diff
@@ -97,12 +97,12 @@ https://grafana.com/docs/mimir/<MIMIR_VERSION>/release-notes/
 
 ## Link to external pages
 
-Use the full URL.
+Use the full URL. Include trailing slashes.
```

Comment:
In this case, you just want to use the URL of the page as it is.
Trailing slashes are a feature of Hugo sites called "pretty URLs" and are not universal on the internet.

Reply:
What about?
```suggestion
Use the full URL. Include trailing slashes where they are present. If a trailing slash isn't present when you copy the URL, don't add one.
```

Reply:
@jdbaldry What if we had two examples for this? One with and one without a trailing slash? Also when I go to github.com and copy the full URL, it includes a trailing slash.

Reply:
The github.com one is interesting because it does indeed copy with a trailing slash.

I think the core of the advice is just copy the URL as it is.

If you go to https://github.com and copy the address bar you get `https://github.com/`
If you go to https://github.com/grafana/grafana and copy the address bar you get `https://github.com/grafana/grafana`.

Reply:
Right, but the thing that people are unsure about is the trailing slash, so I want to speak to that.

Reply:
What about
```suggestion
Use the full URL. Copy the URL as it is from the address bar. If it includes a trailing slash, include it; if it doesn't, don't.
```